tests/integration/ceph_mgr_test.go

```diff
@@ -0,0 +1,249 @@
+/*
+Copyright 2020 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package integration
+
+import (
+	"encoding/json"
+	"fmt"
+	"sort"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/rook/rook/pkg/operator/k8sutil"
+	"github.com/rook/rook/tests/framework/installer"
+	"github.com/rook/rook/tests/framework/utils"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+)
+
+// **************************************************
+// *** Mgr operations covered by TestMgrSmokeSuite ***
+//
+// Ceph orchestrator device ls
+// Ceph orchestrator status
+// Ceph orchestrator host ls
+// Ceph orchestrator create OSD
+// Ceph orchestrator ls
+// **************************************************
+func TestCephMgrSuite(t *testing.T) {
+	if installer.SkipTestSuite(installer.CephTestSuite) {
+		t.Skip()
+	}
+	// Skip this test suite in master and release builds. If there is an issue
+	// running against Ceph master we don't want to block the official builds.
+	if installer.TestIsOfficialBuild() {
+		t.Skip()
+	}
+
+	s := new(CephMgrSuite)
+	defer func(s *CephMgrSuite) {
+		HandlePanics(recover(), s.cluster, s.T)
+	}(s)
+	suite.Run(t, s)
+}
+
+type CephMgrSuite struct {
```

Comment:
How about if we call this the `CephMasterSuite`? Then we can put all the tests in here that will run against the latest ceph master image. Right now you will add the mgr tests. In the future we might also add other tests that need to run against ceph master.

Reply:
Putting together different kind of tests could have this problems:
- Probably we will finish with a mix of tests with very different nature in the same suite.
- In this suite, OSD's are not present when we start the suite (and I need that ). Tests included in this suit should take this into account. This introduces a dependency between new tests an the creation of the OSD's

But as you points, to have a test suite to run tests against Ceph Master is a very good idea. So what if i create a new  "CephMasterSuite"  that starts with OSD's ?

Reply:
Sounds good to keep this specifically for the mgr suite since it isn't creating OSDs. In the future we can create another CephMasterSuite when we have the nightly CI run.